src/__test__/connection/__snapshots__/parseConnectionString.test.ts.snap

```diff
@@ -8,6 +8,8 @@ exports[`connection string parser Should throw on invalid strings esdb://host1;h
 
 exports[`connection string parser Should throw on invalid strings esdb://localhost/&tlsVerifyCert=false 1`] = `"Unexpected \\"&\\" at position 17, expected ?key=value."`;
 
+exports[`connection string parser Should throw on invalid strings esdb://localhost?keepAliveTimeout=please 1`] = `"Unexpected \\"please\\" at position 34, expected Integer."`;
```

Comment:
**SUGGESTION:** Can we add test scenario that checks also `keepAliveInterval`?

Reply:
Added failure case for romans [here](https://github.com/EventStore/EventStore-Client-NodeJS/pull/135/files#diff-513c77abcbb34a10b9a67df5be0345a17d27aebab9b31a0e34e34c65ab7f1a58R530)